### ISSUE TITLE
Version 1.0.3-1 in webjars is broken

### ISSUE THREAD

**Issue by Gonzih** [webjars/fastclick#7]
Hi, seems like link specified in webjars does not exist. Could you please take a look? http://cdn.jsdelivr.net/webjars/fastclick/1.0.3-1/META-INF/resources/webjars/fastclick/1.0.3/fastclick.js results in:

```
Found WebJar (fastclick : 1.0.3-1) but could not find: META-INF/resources/webjars/fastclick/1.0.3-1/META-INF/resources/webjars/fastclick/1.0.3/fastclick.js
```

**Comment by jamesward** [comment]
A fix for this will be released shortly as `1.0.3-2`.

**Comment by Gonzih** [comment]
Thanks!



—
Regards,
Max
